src/style.css

```
@@ -0,0 +1,113 @@
+@import "variables.css";
+@import "normalize.css";
+
+.container {
+  max-width: 200px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: var(--light-color);
+  border: 1px solid var(--light-color-border);
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--second-color);
+}
+
+.card__id {
+  margin: 4px 0 16px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--grey-color);
+  line-height: 14px;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__rewiew-word {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--second-color);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__prace {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__prace-word {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--grey-color);
+
+}
+
+.card__prace-how-much {
+  line-height: 18px;
+  color: var(--second-color);
+  font-weight: 700;
+}
+
+.card__bottom-link {
+  margin-top: 16px;
+  display: block;
+  text-decoration: none;
+  color: var(--light-color);
+  padding: 12px 56px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  
+  text-transform: uppercase;
+  background: var(--gener-color);
+  border-radius: 5px;
+}
+
+.card__bottom-link:hover {
+  border: 1px solid var(--gener-color);
+  color: var(--gener-color);
+  border-radius: 5px;
+  background-color: var(--light-color);
+  transition: 0.4s;
```

- add transparent border by default and change border color on hover
- add transition property for element (not on hover). If you add transition on hover, your style add smoothly but remove quickly
- always add transition-property. By default, it's `all` and the browser looks at all properties for its element
- your border-radius doesn’t change on hover

```suggestion
  background: var(--gener-color);
  border-radius: 5px;
  border: 1px solid transparent;
  transition: color 0.3s, background-color 0.3s, border-color 0.3s;
}

.card__bottom-link:hover {
  border-color: var(--gener-color);
  color: var(--gener-color);
  background-color: var(--light-color);
```